src/FileHelpers.jl

```diff
@@ -25,4 +31,76 @@ function find_notebook_files_recursive(start_dir)
     # reverse alphabetical order so that week5 becomes available before week4 :)
     reverse(not_interal_notebook_files)
 end
-end
\ No newline at end of file
+
+function generate_static_export(path, settings, original_state=nothing, output_dir=".", jl_contents=nothing)
```

Comment:
This should not be in FileHelpers.jl because it is a core function, not a helper